### ISSUE TITLE
No mention of libgmp3-dev dependency

### ISSUE THREAD

**Issue by borninla** [uwplse/herbgrind#47]
When making Herbgrind on a fresh install of Ubuntu 16.04, I was getting the following error:

```
checking for __gmpz_init in -lgmp... no
configure: error: libgmp not found or uses a different ABI (including static vs shared)
```

A fix for this was simply running ```sudo apt-get install libgmp3-dev```.

**Comment by HazardousPeach** [comment]
Huh, you really shouldn't need a system install of GMP, because Herbgrind unpacks and installs it's own version. Could you post the full makefile output?

**Comment by borninla** [comment]
Sure, I've included a txt of the make results here: [make_results.txt](https://github.com/uwplse/herbgrind/files/3352800/make_results.txt)

**Comment by HazardousPeach** [comment]
Hmm. So I'm not quite sure what's going wrong, and having a little trouble reproducing your issue since a lot depends on gmp on my system. Will continue to look into it.

**Comment by HazardousPeach** [comment]
This issue is fixed in 1867ffb
